### PR TITLE
Adds legion support roles and QoL improvements

### PR DESCRIPTION
**Pull request by rhicora**
- **Legion Explorers** are similar to Veterans, but with a Rangemaster and Binocs, no sidearm
- **Legion Scouts** are similar to Legionaries, but with a Scoped Remington 700, no sidearm
- **Legion Camp Followers** are loyal laborers, they are assigned a random set of worker equipment

- Adds face bandanas to regular legion (Legionaries get black, Vet and Decanus get red)
- Adds binoculars as a usable item, they're like scopes without a gun
- Decanus officers now report to the Centurion instead of Vex
- Reduced Vex spawn rate from 3 to 2

### CODE REVIEW

File: code/modules/jobs/job_types/legion.dm [hunk 1]
```diff
@@ -246,4 +250,141 @@ Legionary
 		/obj/item/reagent_containers/pill/patch/healingpowder=2, \
 		/obj/item/ammo_box/m44=2, \
 		/obj/item/flashlight/flare/torch=1)
-	box = null
\ No newline at end of file
+	box = null
+
+/datum/job/f13explorer
+	title = "Explorer"
```
Comment (LetterN) [inline code/modules/jobs/job_types/legion.dm]: can you add this to the job console blacklist

File: code/modules/jobs/job_types/legion.dm [hunk 2]
```diff
@@ -246,4 +250,141 @@ Legionary
...
+		/obj/item/claymore/machete=1, \
+		/obj/item/ammo_box/magazine/internal/rangemaster=2, \
+		/obj/item/reagent_containers/pill/patch/healingpowder=2, \
+		/obj/item/flashlight/flare/torch=1)
+	box = null
+	r_pocket = /obj/item/binocs
+
+/datum/job/f13scout
+	title = "Scout"
+	flag = F13SCOUT
```
Comment (LetterN) [inline code/modules/jobs/job_types/legion.dm]: and this

File: code/modules/jobs/job_types/legion.dm [hunk 3]
```diff
@@ -246,4 +250,141 @@ Legionary
...
+	backpack_contents = list(
+		/obj/item/restraints/legcuffs/bola/tactical=1, \
+		/obj/item/claymore/machete=1, \
+		/obj/item/reagent_containers/pill/patch/healingpowder=2, \
+		/obj/item/ammo_box/magazine/internal/boltaction=2, \
+		/obj/item/flashlight/flare/torch=1)
+	box = null
+
+/datum/job/f13campfollower
+	title = "Camp Follower"
```
Comment (LetterN) [inline code/modules/jobs/job_types/legion.dm]: and also this

File: code/modules/jobs/job_types/legion.dm (hunk 1 above)
Comment (rhicora) [reply code/modules/jobs/job_types/legion.dm]: Done.

File: code/modules/jobs/job_types/legion.dm (hunk 2 above)
Comment (rhicora) [reply code/modules/jobs/job_types/legion.dm]: Done.

File: code/modules/jobs/job_types/legion.dm (hunk 3 above)
Comment (rhicora) [reply code/modules/jobs/job_types/legion.dm]: Done.

**Comment by AndrewL97** [comment]
Issue found with running build_zooming, will fix in separate pr for swiftness

**Pull request merged**